### PR TITLE
#8700 Add strings for v35, add MARK to existing section

### PR DESCRIPTION
**Pull request by electricRGB**
The strings added for v35 are in their own section at the bottom of the strings file. I will update this in a future PR as I'm updating and re-organizing the strings file as part of this release.

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -640,7 +638,7 @@ extension Strings {
     public static let TrackerProtectionAlertButton =  MZLocalizedString("Settings.TrackingProtection.Alert.Button", value: "OK, Got It", comment: "Dismiss button for the tracker protection alert.")
 }
 
-// Tracking Protection menu
+// MARK: - Tracking Protection men
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:641]: You probably meant "menu" 😅

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -609,13 +607,13 @@ extension Strings {
     public static let AppMenuShowPageSourceString = MZLocalizedString("Menu.PageSourceAction.Title", tableName: "Menu", value: "View Page Source", comment: "Label for the button, displayed in the menu, used to show the html page source")
 }
 
-// Snackbar shown when tapping app store link
+// MARK: - Snackbar shown when tapping app store lin
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:610]: "Link"?

File: Client/Frontend/Strings.swift [hunk 3]
```diff
@@ -697,14 +695,14 @@ extension Strings {
     public static let TPMoreInfo = MZLocalizedString("Settings.TrackingProtection.MoreInfo", value: "More Info…", comment: "'More Info' link on the Tracking Protection settings screen.")
 }
 
-// Location bar long press menu
+// MARK: - Location bar long press men
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:698]: "menu"?

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (electricRGB) [reply Client/Frontend/Strings.swift:641]: LOL. Yep. Why did these get cut off! Weird!

File: Client/Frontend/Strings.swift (hunk 2 above)
Comment (electricRGB) [reply Client/Frontend/Strings.swift:610]:
```suggestion
// MARK: - Snackbar shown when tapping app store link
```

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (electricRGB) [reply Client/Frontend/Strings.swift:641]:
```suggestion
// MARK: - Tracking Protection menu
```

File: Client/Frontend/Strings.swift (hunk 3 above)
Comment (electricRGB) [reply Client/Frontend/Strings.swift:698]:
```suggestion
// MARK: - Location bar long press menu
```

File: Client/Frontend/Library/HistoryPanel.swift [hunk 4]
```diff
@@ -522,7 +522,7 @@ class HistoryPanel: SiteTableViewController, LibraryPanel {
         if indexPath.section == Section.additionalHistoryActions.rawValue {
             return []
         }
-        let title: String = .HistoryPanelDelete
+        let title: String = Strings.HistoryPanelDelete
```
Comment (nishant2718) [inline Client/Frontend/Library/HistoryPanel.swift:525]: You can probably leave out the `: String`

**Review by nishant2718 — APPROVED** [review]
Small nits, but 🚢 🚢 🚢 !!

**Comment by isabelrios** [comment]
PR coming to skipt the part that is making the test AuthenticationTest.testPromptPassCodeUponReentry() fails, please don't be blocked due to that

**Comment by electricRGB** [comment]
Intermittent test failing on BR - but green across the board otherwise. Merging due to time constraints for landing strings.

File: Client/Frontend/Library/HistoryPanel.swift (hunk 4 above)
Comment (electricRGB) [reply Client/Frontend/Library/HistoryPanel.swift:525]: With BR being weird, I'll fix this in the string cleanup PR. Good point, though. :)

**Pull request merged**